src/host/polyfills/simpleView.ts

```diff
@@ -1,10 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import ToolsHost from "../toolsHost";
+
+declare var InspectorFrontendHost: ToolsHost;
+
+interface IRevealable {
+    lineNumber: number;
+    columnNumber: number;
+    uiSourceCode: {
+        _url: string;
+    };
+}
+
+export function revealInVSCode(revealable: IRevealable | undefined, omitFocus: boolean) {
+    if (revealable && revealable.uiSourceCode && revealable.uiSourceCode._url) {
+        InspectorFrontendHost.openInEditor(
+            revealable.uiSourceCode._url,
+            revealable.lineNumber,
+            revealable.columnNumber,
+        );
+    }
+
+    return Promise.resolve();

```

Comment:
nit: make function async and remove return.

Reply:
Due to the post build patching mechanism we are using in this extension (see other comments in this PR) I need to return a promise here.